README.md

```diff
@@ -80,9 +80,9 @@ JNA Platform
 
 [![Maven Central](https://img.shields.io/maven-central/v/net.java.dev.jna/jna-platform.svg?label=Maven%20Central)](https://search.maven.org/artifact/net.java.dev.jna/jna-platform/5.12.1/jar)&nbsp;[jna-platform-5.12.1.jar](https://repo1.maven.org/maven2/net/java/dev/jna/jna-platform/5.12.1/jna-platform-5.12.1.jar)&nbsp;[jna-platform-jpms-5.12.1.jar](https://repo1.maven.org/maven2/net/java/dev/jna/jna-platform-jpms/5.12.1/jna-platform-jpms-5.12.1.jar)
 
-This artifact holds cross-platform mappings and mappings for a number of commonly used platform
+This artifact holds cross-platform mappings and mappings for several commonly used platform
```

Comment:
I think the exact number doesn’t matter and prefer “a number of” here.  See https://english.stackexchange.com/a/42823